mirror/Cargo.toml

```diff
@@ -49,4 +53,7 @@ mc-util-build-script = { path = "../mobilecoin/util/build/script" }
 cargo-emit = "0.2.1"
 # Even though this is unused, it needs to be here otherwise Cargo brings in some weird mixture of packages/features that refuses to compile.
 # Go figure ¯\_(ツ)_/¯
-serde = { version = "1", default-features = false, features = ["alloc", "derive"] }
+serde = { version = "1", default-features = false, features = [
+    "alloc",
+    "derive",
+] }
```

Comment:
Nit: We haven't been putting new lines in our dependencies, not sure what caused this change

Reply:
linter did this automatically

Reply:
I can try to revert and see if it cares